### PR TITLE
Delete global documents and `launch()` method

### LINKED ISSUE #113: Combiner.kt:30: get rid of the global variable and fix...
The puzzle `108-37d99e1c` from #108 has to be resolved: 

https://github.com/objectionary/ddr/blob/480a8ab879867dcc37bc92ca139a22cadd2bbe6d/src/main/kotlin/org/objectionary/ddr/launch/Combiner.kt#L30-L30

The puzzle was created by @c71n93 on 20-Apr-23. 

Estimate: 120 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

### PR DESCRIPTION
**Pull request by c71n93**
Closes #113.

---

## PR-Codex overview
This PR replaces the `DdrLaunched` class with `DdrWorkflow`, which is a renamed and refactored version. Additionally, it updates all references to the renamed class and removes unused imports.

### Detailed summary
- Replaced `DdrLaunched` with `DdrWorkflow`
- Updated all references to the renamed class
- Removed unused imports

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by c71n93** [comment]
@mximp take a look, please

File: src/main/kotlin/org/objectionary/ddr/launch/Main.kt [hunk 1]
```diff
@@ -29,6 +29,5 @@ package org.objectionary.ddr.launch
  * - args[0] path to the folder with the program to be analyzed
  */
 fun main(args: Array<String>) {
-    // launch("/home/olesya/huawei/ssh-ddr/ddr/src/test/resources/unit/in/condition_to_name")
-    launch(path = args[0])
+    DdrLaunched(path = args[0]).launch()
```
Comment (mximp) [inline src/main/kotlin/org/objectionary/ddr/launch/Main.kt:32]: @c71n93 Maybe not the best naming here. Why would you want to "launch()" something that's already "DdrLaunched"? Maybe we could rename `DdrLaunched` into `DdrWorkflow` or `DdrAnalysis` in which case `DdrWorkflow().lanuch()` would look natural.

**Comment by c71n93** [comment]
@mximp check, please

**Comment by c71n93** [comment]
@yegor256 take a look, please

**Pull request merged**